app/screens/settings/notification_settings_mobile/index.js

```diff
@@ -1,19 +1,36 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {bindActionCreators} from 'redux';
 import {connect} from 'react-redux';
 
 import {getConfig} from 'mattermost-redux/selectors/entities/general';
-
 import {getTheme} from 'mattermost-redux/selectors/entities/preferences';
+import {updateMe} from 'mattermost-redux/actions/users';
 
 import NotificationSettingsMobile from './notification_settings_mobile';
+import {getCurrentUser} from 'mattermost-redux/selectors/entities/users';
```

Comment:
This is just some nitpicking, but we generally try to keep imports in alphabetical order based on the import path with anything relative coming last like this:
```javascript
import {getConfig} from 'mattermost-redux/selectors/entities/general';
import {getTheme} from 'mattermost-redux/selectors/entities/preferences';
import {getCurrentUser} from 'mattermost-redux/selectors/entities/users';

import NotificationSettingsMobile from './notification_settings_mobile';
```

Reply:
Sure, just sent the fix. Let me know if it's ok.

Reply:
Yep, looks good